### PR TITLE
Bug/1616 thread found not alive, exiting cygnus

### PR DESCRIPTION
**Pull request by pmo-sdr**
Fixed problem skipping Jetty's Threads in YAFS validation.
#1616

### CODE REVIEW

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/nodes/CygnusApplication.java [hunk 1]
```diff
@@ -461,13 +461,16 @@ private void stopSinks() {
         public void run() {
             Set<Thread> threadSet = Thread.getAllStackTraces().keySet();
             Thread[] threadArray = threadSet.toArray(new Thread[threadSet.size()]);
+
+            // Regex matching Jetty thread names, like: qtp586434923-27 or 106024875@qtp-1176250385-1
+            String jettyThreadNamePattern = "^(\\d+@)?qtp-?\\d{2,}-\\d+";
```
Comment (fgalan) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/nodes/CygnusApplication.java]: CHANGES_NEXT_RELEASE entry for this bugfix should be provide, please.

**Comment by AlvaroVega** [comment]
LGTM. A test case, if possible, would be desirable.

**Comment by fgalan** [comment]
So waiting to feedback from @pmo-sdr about adding unit tests before merging.

**Comment by pmo-sdr** [comment]
Added test cases for YAFS :sunglasses:

**Pull request merged**